tests/nn_test.py

```diff
@@ -720,6 +720,19 @@ def f(hx, _):
     with jax.checking_leaks():
       fwd()  # doesn't crash
 
+  def testLog1mExp(self):
+    self.assertEqual(nn.log1mexp(0), -jnp.inf)
+    self.assertEqual(nn.log1mexp(1), -0.45867515)
```

Comment:
We should avoid equality assertions for floating point values, due to expected differences in rounding error across backends and implementations.